3ev/developer-test
# ISSUE1 - Changed SORT_STRING to SORT_NATURAL to fix order issue

**Change Description**
- File - public/index.php
- Desc - Changed array_multisort 3rd parameter from SORT_STRING to SORT_NATURAL

---

**Testing**
Steps:
- Open browser
- Navigate to index page
- Verify that the listed episodes appear in ascending order

---

**Deployment steps**
- Merge branch 'ScruffyPigeon/developer-test' - `ISSUE1` into `ScruffyPigeon/developer-test' - 'master`
